configs/projects/bolt-runtime.rb

```diff
@@ -122,6 +122,12 @@
     proj.component 'rubygem-ed25519'
   end
 
+  # Building native extensions for the x25519 gem is only supported
+  # on 64-bit, non-Windows systems.
+  unless platform.is_windows? || %w[el6 ubuntu16.04 debian8].include?(platform.dist)
```

Comment:
Can you add this logic to the bolt-server project as well? (just for ubuntu and el6 bc we only build for master platforms)

Reply:
`x25519` isn't compatible with jruby, so the team thought it shouldn't be included in bolt-server.

Reply:
Bolt server uses mri ruby? Is that like a long term "if we move bolt-server to the JVM" thing?

Reply:
Yes. If the Skeletor team would like this added to bolt-server anyways I can update the PR.

Reply:
meh, probs not worth it. I dont know the timeline for moving to JVM which means it probably not soon.